app/models/vendor.rb

```diff
@@ -0,0 +1,17 @@
+class Vendor < ActiveRecord::Base
+  belongs_to :market
+  has_many :products
+  has_many :sales
+
+  def self.current_month
+    Time.now.month
+  end
```

Comment:
This should probably go in a helper method if you were using it still. It doesn't have anything to do with `Vendor` other than it being helpful to have. Generally only things that deal with a model's data should go in this class.